libraries/standard/mqtt/integration-test/test_config.h

```diff
@@ -50,22 +50,11 @@
 /**
  * @brief MQTT server host name.
  *
- * This test uses the Mosquitto test server. This is a public MQTT server; do not
- * publish anything sensitive to this server.
- * Mosquitto MQTT broker can run locally as an alternate option. Please refer to
- * the instructions in https://mosquitto.org/ for running a Mosquitto broker
- * locally.
- */
-#ifndef BROKER_ENDPOINT
-    #define BROKER_ENDPOINT    "test.mosquitto.org"
-#endif
-
-/**
- * @brief Length of MQTT server host name.
+ * @note If using AWS IoT Core, the endpoint can be found in the AWS IoT console
```

Comment:
Can the AWS IoT Core be used for all of our integration tests? Don't we have tests for the retain flag, which AWS IoT doesn't support?

Reply:
Good point, have removed all comments pointing to use of AWS IoT from the configs